docs/pages/reference/networking.mdx

```diff
@@ -36,21 +36,9 @@ following use cases:
 </TabItem>
 <TabItem scope={["cloud", "team"]} label="Cloud-Hosted">
 
-All Teleport services (e.g., the Application Service and Database Service) have
-an optional `public_addr` property that you can modify in each service's
-configuration file. The public address can take an IP or a DNS name. It can also
-be a list of values:
-
-```yaml
-public_addr: ["service-one.example.com", "service-two.example.com"]
-```
+On Teleport Enterprise (Cloud) the Teleport agent services always
+connect using reverse tunnels so there is no need to set a public address for a agent.
```

Comment:
```suggestion
connect using reverse tunnels so there is no need to set a public address for an Agent.
```
Grammar fix. See: https://www.britannica.com/dictionary/eb/qa/how-do-you-know-whether-to-use-a-or-an